### PR TITLE
1 Week and 1 Repo Voting Requirements

### PR DESCRIPTION
**Pull request by MegosAlpha**
1 Month means that not much of the community gets to join the project. Not many users are spam accounts, so we may as well increase the number of new users then restrict it to older users only.

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
 AFTER_HOURS_END = 10
 
 # how old do voters have to be for their vote to count?
-MIN_VOTER_AGE = 1 * 30 * 24 * 60 * 60  # 1 month
+MIN_VOTER_AGE = 12 * 60 * 60  # 1 day (12 hrs)
```
Comment (phil-r) [inline settings.py]: soooo 1 day or 12 hours?

Comment (MegosAlpha) [reply settings.py]: @phil-r 1 day, not 1 day and 1 night. I just wanted to be clear.

**Comment by syvb** [comment]
👍 1 week sounds good; this project hasn't been around for more than a week, so spam wouldn't be an issue.
~~12 hours is too little time. I could create 50 accounts right now, and in the morning I could spam any PR I wanted with up/downvotes.~~

File: settings.py (hunk 1 above)
Comment (phil-r) [reply settings.py]: there were 24 hours in the day last time I checked...

Comment (MegosAlpha) [reply settings.py]: @phil-r Yeah, I see how that would be confusing. It's 1 week now, so no worries.

**Comment by rhengles** [comment]
We need to filter serious users from spam accounts. These users need to be checked for activity somehow, be It repos, commits, or something. They must absolutely not be empty accounts.

**Comment by MegosAlpha** [comment]
@rhengles User now must have at least one public repo before voting.

**Comment by PlasmaPower** [comment]
@MegosAlpha that's easy to create

**Comment by rhengles** [comment]
Yes... I don't think there's an algorithm that can reliably tell users from spam accounts apart. I'd approve a manual whitelist of new users to be added through votes in PRs.

**Comment by phil-r** [comment]
Not sure if someone plans an attack or it's actually important 🤔

**Comment by rudehn** [comment]
So why are PRs being marked as having all checks passed even when they don't have the required number of votes?

**Comment by PlasmaPower** [comment]
@rudehn Chaosbot is down, so its status isn't showing up. That leaves only Travis, which is passing.

**Comment by rudehn** [comment]
ahh thanks! Makes perfect sense

**Pull request closed without merging**